test/acceptance/partner_profile.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-env mocha */
+import { setup, teardown } from './helpers'
+import { server as antigravity } from 'antigravity'
+
+describe('Partner profile page', () => {
+  let gravity, browser
+
+  before(async () => {
+    ({ gravity, browser } = await setup())
+    gravity.use(antigravity)
```

Comment:
Got it. But this isn't strictly required as part of the new setup, right? (sorry for the n00b questions, a lot of this is still magical to me)